packages/shared/src/components/MainFeedLayout.tsx

```diff
@@ -247,11 +254,27 @@ export default function MainFeedLayout({
 
   const FeedPageComponent = shouldUseFeedLayoutV1 ? FeedPageLayoutV1 : FeedPage;
 
+  const { data: isSearchPanelGradientActive } = useQuery(
+    searchPanelGradientQueryKey,
+    () => queryClient.getQueryData(searchPanelGradientQueryKey) ?? false,
+    disabledRefetch,
+  );
+
   return (
     <FeedPageComponent
       className={classNames('relative', shouldUseFeedLayoutV1 && '!pt-0')}
     >
-      <div id={searchPanelGradientElementId} />
+      {isSearchPanelGradientActive && (
+        <FeedGradientBg
+          className={classNames(
+            'right-0 -z-1 hidden translate-x-0 opacity-0 transition-opacity duration-500 laptop:flex',
+            sidebarExpanded
+              ? '!-left-32 !max-w-[calc(100%)]'
```

Comment:
Why do you need the calc 100% here?

Reply:
Good catch, I think I was playing with values and then left it.